editor/src/messages/tool/tool_messages/path_tool.rs

```diff
@@ -140,6 +142,7 @@ impl Fsm for PathToolFsmState {
 						tool_data.overlay_renderer.render_subpath_overlays(&document.document_legacy, layer_path.to_vec(), responses);
 					}
 
+					tool_data.opposing_handle_lengths = None;
```

Comment:
I reset the `opposing_handle_lengths` for multiple messages (like when the selected points change). I'm not 100% sure whether I've covered all the correct messages.

Reply:
Looks fine to me.